src/PVRIptvData.cpp

```diff
@@ -802,7 +802,17 @@ bool PVRIptvData::LoadPlayList(void)
       channel.iTvgShift         = tmpChannel.iTvgShift;
       channel.bRadio            = tmpChannel.bRadio;
       channel.properties        = tmpChannel.properties;
-      channel.strStreamURL      = strLine;
```

Comment:
You need to keep this line. Otherwise you will never set the stream url if you are not setting user agent.

Reply:
`else
{
  channel.strStreamURL = strLine;
}

Reply:
That will work too.